superset-frontend/src/dashboard/components/Header/index.jsx

```diff
@@ -138,7 +137,8 @@ const actionButtonsStyle = theme => css`
   }
 `;
 
-const StyledUndoRedoButton = styled(AntdButton)`
+const StyledUndoRedoButton = styled(Button)`
+  // TODO: check if we need this.
```

Comment:
I think it's used here ->
<img width="306" alt="Screenshot 2024-08-19 at 4 20 58 PM" src="https://github.com/user-attachments/assets/93215109-d98d-48b4-852d-0a665a17e9d2">

Reply:
oh, I meant the CSS below it ;) It might not need those overrides... but I'll circle back to that.